conf/snapshot_dir_conf.py

```diff
@@ -3,4 +3,4 @@
 """
 import os
 
-snapshot_dir = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),'utils', 'snapshot')
+snapshot_dir = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),'conf', 'snapshot')
```

Comment:
@mohanqxf2 What's the use of this path? We can use this in the test right instead of declaring it again

Reply:
I have updated the test script to use the path form this conf